packages/analytics-js-integrations/__tests__/integrations/GA4/browser.test.js

```diff
@@ -52,18 +52,23 @@ describe('Google Analytics 4 Track events tests', () => {
 });
 
 describe('Google Analytics 4 Page events tests', () => {
-  let ga4;
-  beforeEach(() => {
-    ga4 = new GA4(
-      { measurementId: 'G-123456', extendPageViewParams: true, capturePageView: 'rs', piiPropertiesToIgnore: [{ piiProperty: '' }] },
-      { getUserId: () => '1234', getUserTraits: () => { } },
-      destinationInfo,
-    );
-    ga4.init();
-    window.gtag = jest.fn();
-  });
+  pageEvents.forEach((event, index) => {
+    let ga4;
+    beforeEach(() => {
+      ga4 = new GA4(
+        {
+          measurementId: 'G-123456',
+          extendPageViewParams: index !== 0,
```

Comment:
This will make things hard to maintain. It is better to take that from the input.

Reply:
addressed